### PR TITLE
Refacto dashboard: call firebase in Dashboard.tsx (update table & tablerow)

### PR DESCRIPTION
**Pull request by MathieuVce**

### CODE REVIEW

File: common/info.ts [hunk 1]
```diff
@@ -55,4 +55,11 @@ export interface AllState {
     users: IUser[];
     reviews: IReview[];
     movies: IMovie[];
+}
+
+export interface ITMDBMovie {
```
Comment (LoicLissens) [inline common/info.ts:60]: IT ?

File: front/src/components/ScrollView.tsx [hunk 2]
```diff
@@ -22,16 +21,71 @@ export const ScrollView: React.FC<IScrollViewProps> = ({ header, body, setId, ty
 
     return (
         <>
-            <div className="py-4">
+            {/* <div className="col-span-full xl:col-span-6 bg-white shadow-lg rounded-sm border border-slate-200">
```
Comment (LoicLissens) [inline front/src/components/ScrollView.tsx:24]: try not to leave this and if you do it for a good reason , leave a comment :)

File: front/src/routes/Dashboard.tsx [hunk 3]
```diff
@@ -58,10 +72,10 @@ const Dashboard: React.FunctionComponent = () => {
                         </div>
             
                         <div className="grid gap-6 lg:grid-cols-2 md:grid-cols-1 sm:grid-cols-1 mt-6">
-                            <Table title="Latest Items" onClick={() => {navigate("/home/items")}} head={headItems} values={valuesItems} icon={<MdOutlineDashboardCustomize size={40}/>}/>
-                            <Table title="Latest Reviews" onClick={() => {navigate("/home/reviews")}} head={headReviews} values={valuesReviews} icon={<AiOutlineStar size={40}/>}/>
-                            <Table title="Latest Users" onClick={() => {navigate("/home/users")}} head={headUsers} values={valuesUsers} icon={<RiUser3Line size={40}/>}/>
-                            <Table title="Latest Comments" onClick={() => {navigate("/home/comments")}} head={headComments} values={valuesComments} icon={<FaRegCommentDots size={40}/>}/>
+                            <Table title="Latest Items" onClick={() => {navigate("/home/items")}} head={headItems} getInfo={async () => {await dispatch(getMovies())}} values={valuesItems} icon={<MdOutlineDashboardCustomize size={40}/>}/>
```
Comment (LoicLissens) [inline front/src/routes/Dashboard.tsx:75]: use a map here

**Review by LoicLissens — APPROVED** [review]

File: common/info.ts (hunk 1 above)
Comment (MathieuVce) [reply common/info.ts:60]: I pour interface et apres TMDB pour the movie database

File: front/src/routes/Dashboard.tsx (hunk 3 above)
Comment (MathieuVce) [reply front/src/routes/Dashboard.tsx:75]: ok !

File: front/src/components/ScrollView.tsx (hunk 2 above)
Comment (MathieuVce) [reply front/src/components/ScrollView.tsx:24]: c'était pour refaire le responsive sur le tableau

**Review by MathieuVce — COMMENTED** [review]
New fix needs merge

**Pull request merged**